code/index.html

```diff
@@ -1,22 +1,180 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lexend+Giga&family=Noto+Sans+JP:wght@300&family=Oswald&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <header>
+    <img class="logo" src="./images/logo.svg" />
+    <nav>
+        <a class="navink" href="#">About</a>
+        <a class="navlink" href="#">Subscribe</a>
+        <a class="navlink" href="#">Contribute</a>
+        <a class="navlink" href="#">Enjoy</a>
+      </div>
+    </nav>
+  </header>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <!-- BIG NEWS SECTION -->
+    <div class="row">
+      <div class="bignews textbox">
+        <h1>We deliver great news daily!</h1>
+        <div class="button">EXPLORE</div>
+      </div>
+      <div class="bignews object-fit">
+        <a href="#">
+          <img src="./images/2.jpg" alt="broken image" />
```

Comment:
Same thing here with the alt text, try to write descriptive alt texts. This continues throughout your project. A better name for this image would be "Person reading newspaper" 📰✌️